app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="pure-g">
+    <div class="pure-u-1-2 gobierto-data-summary-header">
+      <div class="gobierto-data-summary-header-container">
+        <i class="fas fa-clock" />
+        <span class="gobierto-data-summary-header-container-label">
+          {{ labelUpdated }}
+        </span>
+        <span class="gobierto-data-summary-header-container-text">
+          12 oct 2019
+        </span>
+      </div>
+      <div class="gobierto-data-summary-header-container">
+        <i class="fas fa-calendar" />
+        <span class="gobierto-data-summary-header-container-label">
+          {{ labelFrequency }}
+        </span>
+        <span class="gobierto-data-summary-header-container-text">
+          Anual
+        </span>
+      </div>
+      <div class="gobierto-data-summary-header-container">
+        <i class="fas fa-tag" />
+        <span class="gobierto-data-summary-header-container-label">
+          {{ labelSubject }}
+        </span>
+        <a
+          href=""
+          class="gobierto-data-summary-header-container-text-link"
+        >
+          Urbanismo e infraestructuras
+        </a>
+      </div>
+    </div>
+    <div class="pure-u-1-2">
+      <p class="gobierto-data-summary-header-description">
+        {{ datasetDescription }}
+      </p>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: "Info",
+  data() {
+    return {
+      labelUpdated: '',
+      labelFrequency: '',
+      labelSubject: '',
+      labelDownloadData: '',
+      datasetDescription: 'Este conjuntos de datos contiene el detalle de más de 1.200 elementos para actividades de mayores de la ciudad de Madrid con su tipología y coordenadas. En este portal tambien están disponibles otros.'
```

Comment:
Literal string. Shouldn't be a translated text?

Reply:
It's a fake and temporal string.